source/triggers/aws-eventbridge.txt

```diff
@@ -152,13 +170,358 @@ Procedure
+                           "string"
+                        },
+                        "match": ,
+                        "full_document": false,
+                        "full_document_before_change": false,
+                        "unordered": true
+                     ],
+                     "event_processors": [
+                        "AWS_EVENTBRIDGE": {
+                           "account_id": "string",
+                           "region": "string",
+                           "extended_json_enabled": false
+                        },
+                     ],
+                     "error_handler": [
+                        "enabled": true,
+                        "function_id": "string"
+                     ]
+                     }'                        
+
+         .. step:: Deploy the Draft
+
+            If you created a draft, you can deploy all changes in
+            the draft by sending a ``POST`` request with no body to the
+            :admin-api-endpoint:`Deploy a deployment draft
+            <tag/deploy/operation/adminDeployDraft>` endpoint.
+            If you did not create a draft as a first step, the 
+            individual function and trigger requests deployed automatically.
+
+            .. code-block:: shell
+
+               curl -X POST \
+               'https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/drafts/{draftId}/deployment' \
+               --header 'Content-Type: application/json' \
+               --header 'Authorization: Bearer <access_token>' \
+
+.. _eventbridge-error-handler-parameters:
+
+Error Handler Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default error handler has two parameters: ``error`` and ``changeEvent``.
+
+``error`` 
+`````````
+Has the following two attributes:
+
+- ``code``: The code for the errored EventBridge put request. For a list of 
+  error codes used by the error handler, see the below section. 
+
+- ``message``: The unfiltered error message from an errored EventBridge
+  put request.
+
+``changeEvent``
+```````````````
+
+The requested change to your data made by EventBridge. For more information
+on types of change events and their configurations, see 
+:ref:`Change Event Types <database-events>`.
+
+.. _eventbridge-error-codes:
+
+Error Codes
+~~~~~~~~~~~
+
+If an error was recevied from EventBridge, the event processor will parse the 
+error as either ``DOCUMENT_TOO_LARGE`` or ``OTHER``. This parsed error is passed
+to the error handler function through the ``error`` parameter. 
+
+``DOCUMENT_TOO_LARGE``
+``````````````````````
+If the put entry for an EventBridge trigger event is larger
+than 256 KB, EventBridge will throw an error. The error will contain either:
+
+- `status code: 400 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400>`_ and ``total size of the entries 
+   in the request is over the limit``.
+
+- `status code: 413 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413>`_, 
```

Comment:
both bullet items are formatting oddly. maybe this helps (the 2 underscores)
or maybe it's an indent issue on the lines after the bullet?
```suggestion
- `status code: 413 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413>`__, 
```